src/pages/faq.tsx

```diff
@@ -8,14 +8,19 @@ import Footer from "../components/Footer"
 import { PageTitle } from "../components/Text"
 import { Trans } from 'react-i18next'
 
+// Get from/to dates from environment variables, format using toLocaleDateString
+const startDate = process.env.REACT_APP_START_DATE || ''
+const endDate = process.env.REACT_APP_END_DATE || ''
+const [from, to] = [startDate, endDate].map((date: string) => (new Date(date)).toLocaleDateString())
```

Comment:
If we decide to use git committed constants instead of environment variables:
```suggestion
const START_DATE = 'YYYY-MM-DD' // Replace with real date when ready
const END_DATE = 'YYYY-MM-DD' // Replace with real date when ready
const [from, to] = [START_DATE, END_DATE].map((date: string) => (new Date(date)).toLocaleDateString())
```

Reply:
Let's use the React env variables, could we set them up in the [/src/constants.tsx](https://github.com/zkparty/trusted-setup-frontend/blob/main/src/constants.ts) file and import them in the `faq.tsx`?

Reply:
The IPFS build, at least when it's finalised, needs to be completely deterministic. We can achieve that and still have dates specified in variables. We can either:
- use constants.tsx
- Include a `.env` file in the commit, with `REACT_APP_START_DATE` and `REACT_APP_END_DATE`, and other vars such as the sequencer URL. 

I think the latter might be neater.